### DISCUSSION TITLE
I did everything in the tutorial and relative issues it just will not work

### DISCUSSION THREAD

**Discussion by neilhuang007** [anuraghazra/github-readme-stats#3926] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/a8b8f171-24e4-4073-b56c-fa276ebc041b)
i have the environmental variable set and i have the project set up
![image](https://github.com/user-attachments/assets/7a81cbbb-28e7-45b2-9266-a11d72b4f203)
and i also have switched to my custom domain:
![image](https://github.com/user-attachments/assets/f4c85037-0b2e-4683-9aa5-0f2bc152914e)
i have around 280 commits in private res and the card not showing them, please help
** i have looked at similar issues and they solved it by deploying again or doing what the tutorial says, did not work for me

**Comment by qwerty541** [comment, marked as answer]
Hello @neilhuang007 

Did you set the correct access permissions for your personal access token according to the documentation? Here is the link on our recently updated documentation about configuring both classic and fine-grained tokens https://github.com/anuraghazra/github-readme-stats?tab=readme-ov-file#first-step-get-your-personal-access-token-pat. Please make sure that you have enable all required scopes.
